cmd/syncthing/main.go

```diff
@@ -45,19 +45,13 @@ import (
 )
 
 const (
-	exitSuccess            = 0
```

Comment:
Should probably use [these](https://github.com/syncthing/syncthing/blob/31569debeb56d7e11b3d02577157c520ef0236a7/lib/syncthing/syncthing.go#L50-L55) instead for all of these